extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -95,12 +95,14 @@ export let InteractiveComponentDef;
  *    interactiveEmbeddedComponentState: !InteractiveComponentDef,
  *    mutedState: boolean,
  *    pageAudioState: boolean,
+ *    pagePlayableState: boolean,
  *    pausedState: boolean,
  *    previewState: boolean,
  *    rtlState: boolean,
  *    shareMenuState: boolean,
  *    sidebarState: boolean,
  *    storyHasAudioState: boolean,
+ *    storyHasPlayableState: boolean,
```

Comment:
Okay let's bikeshed... 😆 

The `hasPlayable` makes a ton of sense in the context of this PR. I really warmed up to it as I was reading your code.
I'm still a bit afraid that it will take time and energy for someone new to figure out what this does.
My suggestions:
- `storyHasPlaybackUiState` and `pageHasElement(s)WithPlaybackState`: definitely has my vote, as it's clearly tied to whatever playback UI we're introducing and might build on
- `storyCanBePausedState`
- `storyHasPlayableElementsState`